courseaffils/models.py

```diff
@@ -141,7 +141,9 @@ class CourseInfo(models.Model):
     # for ability to query what courses are going on NOW
     starttime = models.TimeField(null=True, blank=True)
     endtime = models.TimeField(null=True, blank=True)
-    days = models.CharField(max_length=7, null=True, blank=True)  # e.g. 'MWF'
+    days = models.CharField(
+        max_length=7, null=True, blank=True,
+        help_text='e.g. "MTWRF"')
```

Comment:
Yeah, I remembered that too.. I was running this code in mediathread and it didn't complain that I need to add a migration, so I thought I had mis-remembered. When I ran makemigrations, it was created though, which I've added in a new commit.